lab-elizabeth/test/book-route-test.js

```diff
@@ -0,0 +1,319 @@
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+
+      it('should return 404: not found', done => {
+        request.get(`${url}/api/series/${this.tempSeries._id}/book/hippocampus`)
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+
+    });
+
+  });
+
+  describe('testing POST requests', function(){
+
+    describe('with valid series.id and book.body', () => {
+
+      before(done => {
+        new Series(exampleSeries).save()
+        .then(series => {
+          this.tempSeries = series;
+          done();
+        })
+        .catch(done);
+      });
+
+      after(done => {
+        exampleBook.published = '2005';
+        exampleBook.author = 'R.F.Rankin';
+        exampleBook.title = 'The Brightonomicon';
+        exampleBook.description = 'Hugo Rune vs. Count Otto Black. Rizla cannot remember any of it';
+        Promise.all([
+          Series.remove({}),
+          Book.remove({}),
+        ])
+        .then(() => done())
+        .catch(done);
+      });
+
+      it('should create a book', done => {
+        request.post(`${url}/api/series/${this.tempSeries._id}/book`)
+        .send(exampleBook)
+        .end((err, res) => {
+          if(err) return done(err);
+          expect(res.status).to.equal(200);
+          expect(res.body.seriesID).to.equal(this.tempSeries._id.toString());
+          done();
+        });
+      });
+
+      it('should create a book', done => {
+        exampleBook.published = '';
+        request.post(`${url}/api/series/${this.tempSeries._id}/book`)
+        .send(exampleBook)
+        .end((err, res) => {
+          if(err) return done(err);
+          expect(res.status).to.equal(200);
+          expect(res.body.seriesID).to.equal(this.tempSeries._id.toString());
+          done();
+        });
+      });
+
+    });
+
+    describe('with invalid id or body', () => {
+
+      before(done => {
+        new Series(exampleSeries).save()
+        .then(series => {
+          this.tempSeries = series;
+          done();
+        })
+        .catch(done);
+      });
+
+      after(done => {
+        exampleBook.published = '2005';
+        exampleBook.author = 'R.F.Rankin';
+        exampleBook.title = 'The Brightonomicon';
+        exampleBook.description = 'Hugo Rune vs. Count Otto Black. Rizla cannot remember any of it';
+        Promise.all([
+          Series.remove({}),
+          Book.remove({}),
+        ])
+        .then(() => done())
+        .catch(done);
+      });
+
+      it('should return 404: not found', done => {
+        request.post(`${url}/api/series/hippocampus/book`)
+        .send(exampleBook)
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+
+      it('should return 404: not found', done => {
+        request.post(`${url}/api/series/hippocampus/book`)
```

Comment:
It's good that you're being thorough, but you don't have to test a 404 response for every request method :)